app/src/test/java/mozilla/lockbox/store/RouteStoreTest.kt

```diff
@@ -43,9 +84,37 @@ class RouteStoreTest {
 
     @Test
     fun `dispatched non-routeactions`() {
+        setUpSubject()
         val action = LifecycleAction.UserReset
         dispatcher.dispatch(action)
 
         routeObserver.assertEmpty()
     }
+
+    @Test
+    fun `datastore failed to create backend`() {
+        setUpErrorSubject()
+
+        dispatcher.dispatch(DataStoreAction.Sync)
```

Comment:
I would rather see a test that pushes out new `DataStore.State`s.

Reply:
Done!